### PR TITLE
added archive and trash functionnality

### PR DESCRIPTION
**Pull request by cjinc121**
- added trash
- added archive
- added pin Unpin 
 features
Screenshot (140)
Screenshot (141)
Screenshot (142)

### CODE REVIEW

File: notify/src/component/note-display/noteDisplayCard.jsx [hunk 1]
```diff
@@ -1,35 +1,138 @@
...
-{
-notes.map((note)=>{
-  return <div className="note-display-card" >
-  <div className="note-display-title"> 
-<p>{note.title}</p><BsFillPinFill /></div>
-<div className="note-display-body"><p> {note.body}</p></div>
-<div className="note-display-footer"> 
-<IoColorPaletteOutline/>  
-<MdOutlineArchive /> 
-<MdDelete onClick={()=>{setNotes((notes)=>notes.filter((filterNote)=>filterNote.id!=note.id))}}/>
```
Comment (psychoactiv) [inline notify/src/component/note-display/noteDisplayCard.jsx:24]: suggestion: Keep lengthy functions outside of the render

**Review by krishanu2911 — APPROVED** [review]
No changes are required Well written code
-Nice descriptive naming 
-Nice use of dispatch
Good to Go👍🏻

File: notify/src/component/page-container/ArchiveDisplayCard.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import { IoColorPaletteOutline } from "react-icons/io5";
+import "../note-display/noteDisplay.css";
+const ArchiveDisplayCard = () => {
+  const { notesState, notesDispatch } = useNotesContext();
+
+  return (
+    <>
+      <div className="note-display-container">
+        <h3>
+          {notesState.notes.filter(
```
Comment (psychoactiv) [inline notify/src/component/page-container/ArchiveDisplayCard.jsx:12]: suggestion : You can destructure the notes in the useNotesContext function call itself that would avoid long chaining.

**Review by psychoactiv — APPROVED** [review]
Code looks well written 
proper naming conventions are folllowed
you could work on the given reviews
Code is ready to merge

**Pull request merged**